plugins/plugin-apache-composer/src/test/composer1/composer-kill.ts

```diff
@@ -28,7 +28,6 @@
 // const killFailure404 = 'Cannot find live session'
 
 import { ISuite } from '@kui-shell/core/tests/lib/common'
-import * as openwhisk from '@kui-shell/plugin-openwhisk/tests/lib/openwhisk/openwhisk'
```

Comment:
openwhisk is actually used in the test content (which has been commented out). If we want to keep this commented content, maybe we should comment out this line instead of remove it. 

@starpit , do we want to keep this test?